app/templates/components/user-cards.hbs

```diff
@@ -1,10 +1,32 @@
 {{!-- {{yield}} --}}
 
 <div class = "container2">
-</div>
-   <div class="content-container">
-      {{#each teamsl as |avatar|}}
-         {{user-card firstname = avatar.first_name lastname = avatar.last_name email = avatar.email image = avatar.image}}
-      {{/each}}
+   <div class = "dropdown-container">
+      <div class = "dropdown-text">Filter By</div>
+      <BsDropdown as |dd|>
+         <dd.toggle><div class ="option-text"> {{this.select_team}} </div>
+            <svg class = "vector" width="8" height="5" viewBox="0 0 8 5" fill="none" xmlns="http://www.w3.org/2000/svg">
+               <path d="M4.00001 4.29169C3.89454 4.29169 3.76748 4.21486 3.68155 4.14064L0.208344 1.58335C0.0286562 1.41929 
+                  0.0416781 0.958349 0.25001 0.750015C0.512442 0.487582 0.903657 0.593768 1.08334 0.753924L3.9961 2.8021L6.70835 
+                  0.541682C6.88803 0.37762 7.28743 0.263214 7.54168 0.541682C7.83015 0.85763 7.72137 1.21096 7.54168 1.37502L4.28907 
+                  4.14064C4.21095 4.21486 4.10548 4.29169 4.00001 4.29169Z" fill="#79888E"/>
+            </svg>
```

Comment:
We can make use of `svg-jar` for this. Refer https://www.npmjs.com/package/ember-svg-jar